### ISSUE TITLE
ServerConsistency 1.4 [Fixed some stuff + Logging!] [NEW UPDATE]

### ISSUE THREAD

**Issue by playingo** [GrowtopiaNoobs/GrowtopiaServer#644]
-- You may know what it does, explained in here: https://github.com/GrowtopiaNoobs/GrowtopiaServer/issues/613 --
Also, this is the first ever "plugin" for Anybody's Growtopia Private Server.
- FIXED CHECKING FOR 90B INSTEAD **90KB**.
- Added logs.

Download: https://github.com/playingo/ServerConsistency/blob/master/ServerConsistency1.4.zip

**Comment by OverallGod** [comment]
Awesome :)

**Comment by playingo** [comment]
Thanks.

**Comment by ilyn1337** [comment]
Deleting 90k worlds is retarded, imagine, someone fully clear his world and file size is 85KB, lol go find some json api to check, not just file size, this application shouldn't be used

**Comment by playingo** [comment]
i have made a test and a fully cleared world is exact 90kb @ilyn1337, also https://www.youtube.com/watch?v=Ul81fVwipyc&feature=youtu.be complete avoid of corrupting

**Comment by GrowtopiaNoobs** [comment]
True, file size can decrease a bit by breaking rocks and cave backgrounds, maybe it can be even that 85kB. Maybe it can be good to add warning about this in readme and maybe add possibility to change value of minimal world size. That full json check would be cool, but idk about speed, when there is alot of files. Maybe it can be added to new version as second option. It shouldn't be hard, just load that json and check if there is exception.

**Comment by playingo** [comment]
well yeah i mean, i have claimed that this is a simple program in one of the first versions which was i believe 1.1 or 1.0 i dont remember anymore. Anyways, it works great since most corrupted files are below 90kb and the chances are REALLY low that you will ever face a corrupted file with a legitimate file size. and incase it does go upto 85 kb, i'll improve it. As I said, this will never be perfect and it is a simple program, but hey, it works and I doubt you will ever get corruptions with this. Chances are too low. Anyways, I'll accept any critism to make it better :D

**Comment by ZentaiZber** [comment]
![image](https://user-images.githubusercontent.com/51924277/60488358-f721ea80-9ccb-11e9-992f-6a3b285128e9.png)
Not Work!

**Comment by playingo** [comment]
@ZentaiZber bruh it restarts every 12 seconds by default and you didnt even start server. make sure server is called "server.exe" and next time tell me what the problem is, don't say "Not Work" only.

**Comment by ilyn1337** [comment]
B R U H

**Comment by playingo** [comment]
bruh moment

**Comment by ZentaiZber** [comment]
bruh i dont know to use haha thx you btw

**Comment by rheein** [comment]
Awesome Plugin, unfortunately this does not support Linux OS :(
